front/pages/api/w/[wId]/assistant/agent_configurations/[aId]/feedbacks.ts

```diff
@@ -0,0 +1,63 @@
+import type { WithAPIErrorResponse } from "@dust-tt/types";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { apiErrorForConversation } from "@app/lib/api/assistant/conversation/helper";
+import type { AgentMessageFeedbackType } from "@app/lib/api/assistant/feedback";
+import { getAgentConfigurationFeedbacks } from "@app/lib/api/assistant/feedback";
+import { withSessionAuthenticationForWorkspace } from "@app/lib/api/auth_wrappers";
+import type { Authenticator } from "@app/lib/auth";
+import { apiError } from "@app/logger/withlogging";
+
+async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<
+    WithAPIErrorResponse<{ feedbacks: AgentMessageFeedbackType[] }>
+  >,
+  auth: Authenticator
+): Promise<void> {
+  if (!(typeof req.query.aId === "string")) {
+    return apiError(req, res, {
+      status_code: 400,
+      api_error: {
+        type: "invalid_request_error",
+        message: "Invalid query parameters, `aId` (string) is required.",
+      },
+    });
+  }
+
```

Comment:
Where is the check that i Do have access to this agent configuration?

Reply:
I think you want to first pull agentConfiguration with getAgentConfiguration which relies on getAgentConfigurations which enforce a lot of safety wrt to spaces.

And then pass a full AgentConfigurationType to getAgentConfigurationFeedbacks

Here you're disclosing feedback I think of agents you don't necessarilly have access to as they may be based on data sources that are in spaces you don't have access to.